### PR TITLE
dkms: update to 518.5

### PR DESCRIPTION
**Pull request by dvrogozh**

### CODE REVIEW

File: doc/virtualization.rst [hunk 1]
```diff
@@ -319,6 +313,41 @@ that such resource allocation will make GPU basically unusable from the host.
...
+    # consider other "*_spare" resources...
+
+* Explicitly allocate resources for each VM::
+
+    sudo su
+    CARD=/sys/class/drm/card1
+
+    echo BYTES > $CARD/prelim_iov/vf1/gt/lmem_quota
+
+Kernel mode driver will reject too high or too low resource values.
```
Comment (dsocek) [inline doc/virtualization.rst:336]: too high or too low resource values ->
resource values that are too high or too low

File: doc/virtualization.rst [hunk 2]
```diff
@@ -319,6 +313,41 @@ that such resource allocation will make GPU basically unusable from the host.
...
+
+Another aspect you might wish to tune for your VMs is scheduling settings. Pay attention on execution
```
Comment (dsocek) [inline doc/virtualization.rst:338]: Pay attention on -> Pay attention to

File: doc/intel-gpu-dkms.rst [hunk 3]
```diff
@@ -163,6 +165,43 @@ Change Log
...
+  the future. Instead, use the following 2 ways to allocate VF resources:
+
+  * Use ``/sys/class/drm/card%/prelim_iov/pf/gt/*_spare`` to specify resources reserved for PF. This will reduce
+    amount of resources you can spend for VFs
+
+  * Use ``/sys/call/drm/card%/prelim_iov/vf%/gt/*_quota`` to explicitly specify amount of resources available
+    for VFs
+
+  You can also rely on kernel mode driver auto provisioning to distribute resources across VMs (which is fair
+  distribution). Consider however to tune execution quantum and preemption timeout settings for the real life
```
Comment (dsocek) [inline doc/intel-gpu-dkms.rst:195]: Consider however -> However, consider

File: doc/intel-gpu-dkms.rst [hunk 4]
```diff
@@ -163,6 +165,43 @@ Change Log
...
+* https://github.com/intel-gpu/intel-gpu-firmware/releases/tag/22WW51.5_518.5
+* And other tags with "518" in the name
+
+Changes:
+
+* ``/sys/class/drm/card%/iov/pf/gt/available/*_quota`` PF attributes got deprecated and might be removed in
+  the future. Instead, use the following 2 ways to allocate VF resources:
+
+  * Use ``/sys/class/drm/card%/prelim_iov/pf/gt/*_spare`` to specify resources reserved for PF. This will reduce
+    amount of resources you can spend for VFs
```
Comment (dsocek) [inline doc/intel-gpu-dkms.rst:189]: the amount

File: doc/intel-gpu-dkms.rst [hunk 5]
```diff
@@ -163,6 +165,43 @@ Change Log
...
+
+  * Use ``/sys/call/drm/card%/prelim_iov/vf%/gt/*_quota`` to explicitly specify amount of resources available
```
Comment (dsocek) [inline doc/intel-gpu-dkms.rst:191]: the amount

**Review by dsocek — APPROVED** [review]
Looks good.  Some small textual changes suggested

File: doc/virtualization.rst (hunk 1 above)
Comment (dvrogozh) [reply doc/virtualization.rst]: done

File: doc/virtualization.rst (hunk 2 above)
Comment (dvrogozh) [reply doc/virtualization.rst]: done

File: doc/intel-gpu-dkms.rst (hunk 3 above)
Comment (dvrogozh) [reply doc/intel-gpu-dkms.rst]: done

File: doc/intel-gpu-dkms.rst (hunk 4 above)
Comment (dvrogozh) [reply doc/intel-gpu-dkms.rst]: done

File: doc/intel-gpu-dkms.rst (hunk 5 above)
Comment (dvrogozh) [reply doc/intel-gpu-dkms.rst]: done

**Pull request merged**